grakn-test/src/test/java/ai/grakn/test/GraknTestEnv.java

```diff
@@ -146,10 +146,12 @@ static String randomKeyspace(){
     }
 
     public static void hideLogs() {
-        ((Logger) org.slf4j.LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME)).setLevel(Level.OFF);
-        ((Logger) org.slf4j.LoggerFactory.getLogger(GraknTestEnv.class)).setLevel(Level.DEBUG);
-        ((Logger) org.slf4j.LoggerFactory.getLogger(BackgroundTaskTestUtils.class)).setLevel(Level.DEBUG);
+        ((Logger) org.slf4j.LoggerFactory.getLogger(ROOT_LOGGER_NAME)).setLevel(Level.ERROR);
+
+        // Some dependencies are using log4j, so must be managed separately
         org.apache.log4j.Logger.getRootLogger().setLevel(org.apache.log4j.Level.ERROR);
+
+        ((Logger) org.slf4j.LoggerFactory.getLogger("ai.grakn")).setLevel(Level.INFO);
```

Comment:
We don't hide `ERROR` logs - it will show every level `INFO` and above (so `INFO`, `WARN` and `ERROR`).

It's true we won't see `DEBUG`. In general, the `DEBUG` level seems to high - I wouldn't be surprised if travis exceeds the log limit. I can try it and see if travis can handle it.